src/main/java/com/systemmeltdown/meltdownlog/topics/DoubleTopic.java

```diff
@@ -0,0 +1,29 @@
+package com.systemmeltdown.meltdownlog.topics;
+
+import com.systemmeltdown.meltdownlog.lib.Utils;
+
+public class DoubleTopic extends Topic {
+	private double m_roundingFactor;
+	private double m_lastRoundedValue = Double.MIN_VALUE;
+	private long m_lastNanos = Long.MIN_VALUE;
+
+	public DoubleTopic(String name, double roundingFactor) {
+		super(name, Double.TYPE);
+		m_roundingFactor = roundingFactor;
```

Comment:
Check that roundingFactor != 0

Reply:
Thanks! This is fixed in the next commit.